content/graphing/faq/as_count_validation.md

```diff
@@ -0,0 +1,19 @@
+---
+title: as_count in monitors
+kind: faq
+---
+
+Datadog’s graphs utilize [time aggregation](/graphing/faq/what-is-the-granularity-of-my-graphs-am-i-seeing-raw-data-or-aggregates-on-my-graph) to reduce the point count on a timeframe; this is done for performance reasons because granularity higher than 350 data-points in a graph doesn't provide additional insight. 
+
+#### What changed? 
+We used to allow to create monitors that use `average`/`min`/`max` monitor aggregation with `as_count` [function](/graphing/miscellaneous/functions).   
```

Comment:
Previously, we allowed the creation of monitors that use

Reply:
👍